contrib/api-stress-test/roles/locust/config/templates/stress.py

```diff
@@ -0,0 +1,100 @@
+import os
+import requests
+import socket
+import yaml
+import uuid
+import jinja2
+from locust import task, between, constant_pacing
+from locust.contrib.fasthttp import FastHttpUser
+
+SERVICE_HOST_ENV_NAME = "KUBERNETES_SERVICE_HOST"
+SERVICE_PORT_ENV_NAME = "KUBERNETES_SERVICE_PORT"
+SERVICE_TOKEN_FILENAME = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+SERVICE_CERT_FILENAME = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+
+
+def _join_host_port(host, port):
+    """Adapted golang's net.JoinHostPort"""
+    template = "%s:%s"
+    host_requires_bracketing = ':' in host or '%' in host
+    if host_requires_bracketing:
+        template = "[%s]:%s"
+    return template % (host, port)
+
+
+def read_template(template_path):
+    with open(template_path, "r") as f:
+        template_data = f.read()
+    return template_data
+
+
+def generate_from_template_dict(template_data, jobname, vc):
+    generated_file = jinja2.Template(template_data).render(
+        {
+            'jobname': jobname,
+            'vc': vc
+        }
+    )
+    return generated_file
+
+
+with open(SERVICE_TOKEN_FILENAME) as f:
+    kube_token = f.read()
+
+kube_cert = SERVICE_CERT_FILENAME
+k8s_headers = {
+    "Authorization": "Bearer {0}".format(kube_token)
+}
+kube_url = "https://{0}".format(_join_host_port(os.environ[SERVICE_HOST_ENV_NAME], os.environ[SERVICE_PORT_ENV_NAME]))
+pai_token = os.environ["PAI_TOKEN"]
+job_template = read_template("/mnt/locust/job.yml")
+
+
+class K8SAgent(FastHttpUser):
+    wait_time = constant_pacing(30)
+    jobid = 0
+    userid = None
+    max_retries = {{ locust_http_max_retries }}
+    connection_timeout = {{ locust_http_connection_timeout }}
+    network_timeout = {{ locust_http_network_timeout }}
+
+    def on_start(self):
+        self.userid = str(uuid.uuid4())
+
+{% if stress_test_openpai_job_submit_enable %}
+    @task({{ test_api_list.openpai_job_submit.weight }})
+    def submitjob(self):
```

Comment:
rename to `submit_job`?

Reply:
Done

Reply:
...I mean the python function name. rename `def submitjob` to `def submit_job`. Same as below

Reply:
Done